book/caching.md

```diff
@@ -1,32 +1,20 @@
+---
+layout: default
+title: Caching
+permalink: /caching
+last_modified_date: '2023-04-13'
+---
+
 # Caching and cache tags
+{: .no_toc .fw-500 }
 
-<h3 style="text-align: center;">
-<a href="/d9book">home</a>
-</h3>
-
-- [Caching and cache tags](#caching-and-cache-tags)
-  - [How to uncache a particular page or node](#how-to-uncache-a-particular-page-or-node)
-  - [Don't cache data returned from a controller](#dont-cache-data-returned-from-a-controller)
-  - [Disable caching for a content type](#disable-caching-for-a-content-type)
-  - [Considering caching when retrieving query, get or post parameters](#considering-caching-when-retrieving-query-get-or-post-parameters)
-  - [Debugging Cache tags](#debugging-cache-tags)
-  - [Using cache tags](#using-cache-tags)
-  - [Setting cache keys in a block](#setting-cache-keys-in-a-block)
-  - [Getting Cache Tags and Contexts for a block](#getting-cache-tags-and-contexts-for-a-block)
-  - [Caching REST Resources](#caching-rest-resources)
-  - [Caching in an API class wrapper](#caching-in-an-api-class-wrapper)
-  - [Caching in a .module file](#caching-in-a-module-file)
-  - [Logic for caching render arrays](#logic-for-caching-render-arrays)
-  - [Development Setup](#development-setup)
-    - [Disable caching and enable TWIG debugging](#disable-caching-and-enable-twig-debugging)
-    - [Disable Cache for development](#disable-cache-for-development)
-  - [Reference](#reference)
+## Table of contents
+{: .no_toc .text-delta }
```

Comment:
Am I understanding correctly that this is a dynamic TOC?

Reply:
Yes, more about this in [this documentation](https://just-the-docs.github.io/just-the-docs/docs/navigation-structure/#in-page-navigation-with-table-of-contents).